uber/templates/preregistration/preregbase.html

```diff
@@ -69,7 +71,7 @@
                 extra: 0
             }]
         };
```

Comment:
That sounds good.  There's another edge case I thought of involving this change, which I just filed at #1274 but my opinion is that we don't need to hold up this PR for that.  (Though if you disagree and/or feel led to test it out now then that's fine too.)